.storybook/preview.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Global, ThemeProvider } from '@emotion/react';
+
+import { reset, theme } from '../src';
+
+// eslint-disable-next-line import/prefer-default-export
+export const decorators = [
```

Comment:
We no longer need to manually add `knobs` or `a11y` decorators; they seem to be included automatically when the addons are configured in `main.js`.